src/Catel.MVVM/Catel.MVVM.Shared/Windows/Helpers/WrapControlHelper.cs

```diff
@@ -225,7 +225,29 @@ public static Grid Wrap(FrameworkElement frameworkElement, WrapOptions wrapOptio
                         button.Command = dataWindowButton.Command;
                     }
 
-                    button.Content = dataWindowButton.Text;
+                    if (dataWindowButton.ContentBindingPath != null)
```

Comment:
Let me think more about this, maybe it's better idea to create this BindingDescription class.